packages/svelte/src/index-client.js

```diff
@@ -17,6 +17,9 @@ import { legacy_mode_flag } from './internal/flags/index.js';
  *
  * `onMount` does not run inside [server-side components](https://svelte.dev/docs/svelte/svelte-server#render).
  *
+ * `onMount` runs only once. If you need reactivity, you can use [`$effect`](https://svelte.dev/docs/svelte/$effect),
```

Comment:
I have some changes in mind here but I'll need to push them up, I can't make a suggestion in the UI